src/man/kiwix-manage.1

```diff
@@ -1,62 +1,72 @@
-.TH KIWIX 1 "21 May 2012"
+.TH KIWIX-MANAGE 1 "21 May 2012"
+
 .SH NAME
 kiwix\-manage \- Kiwix Library Manager
+
 .SH SYNOPSIS
 .IX Header SYNOPSIS
-.B kiwix\-manage
-.br
-kiwix\-manage LIBRARY_PATH add ZIM_PATH ...
-.br
-	kiwix-manage LIBRARY_PATH show [CONTENTID1] [CONTENTID2] ...
-.br
-	kiwix\-manage LIBRARY_PATH remove CONTENTID1 [CONTENTID2]
+.TP
+\fBkiwix\-manage\fR LIBRARY_PATH add ZIM_PATH ...
+.TP
+\fBkiwix\-manage\fR LIBRARY_PATH show [ZIM_ID_1] [ZIM_ID_2] ...
+.TP
+\fBkiwix\-manage\fR LIBRARY_PATH remove ZIM_ID_1 [ZIM_ID_2] ...
+.TP
+\fBkiwix\-manage\fR --version
+.TP
+\fBkiwix\-manage\fR --help
+
 .SH DESCRIPTION
-.SS kiwix\-manage
-.PP
-\fBkiwix\-manage\fP tool for managing a kiwix library.
 .PP
-Allows one to manage the content of the kiwix library. The library file is a
-flat XML file listing ZIM files with all necessary information like favicon,
-date, creator, description, indexpath, filepath, title, source/metalink, etc.
-.
+\fBkiwix\-manage\fP is a command line tool for manipulating a Kiwix XML library.
 .PP
- Example library file can be found at https://www.kiwix.org/library\.xml
-.br
+\fBkiwix\-manage\fP allows to manage the entries of the Kiwix
+library. The library file is a flat XML file listing ZIM files with
+all necessary information like id, favicon, date, creator,
+description, filepath, title, url, etc.
+
+.SH ACTIONS
+
 .TP
 \fBadd\fR
-Add \fBZIM_FILE\fP to \fBLIBRARY_FILE\fP.
+Add \fBZIM_FILE\fP to \fBLIBRARY_FILE\f.
+
+.TP
+\fBremove\fR
+Remove the given \fBZIM_ID\fR from \fBLIBRARY_FILE\fR. At least one \fBZIM_ID\fR should be specified.
 
 .TP
 \fBshow\fR
-Show given \fBCONTENT_ID\fP from \fBLIBRARY_FILE\fR. If no \fBCONTENT_ID\fP is given then all contents from \fBLIBRARY_FILE\fR are shown.
+Show given \fBZIM_ID\fP from \fBLIBRARY_FILE\fR. If no \fBZIM_ID\fP is given then all contents from \fBLIBRARY_FILE\fR are shown.
 
+.SH OPTIONS
 .TP
-\fBremove\fR
-Remove the given \fBCONTENT_ID\fR from \fBLIBRARY_FILE\fR. At least one \fBCONTENT_ID\fR should be specified.
+Options to be used with the action \fBadd\fR:
 
 .TP
-\fB\-\-zimPathToSave=\fR
-Change the location reference of the ZIM file in the library.
+\fB\-\-url=HTTP_URL\fR
```

Comment:
What about displaying `HTTP_URL` in regular or italic font?

Reply:
Not really sure about that... if OK for you would prefer (at least for the moment) to keep it like this.

Reply:
It's OK